let a = ''

try {
    throw new Error('dkkd')
} catch (error) {
    console.log(error.message)
}